packages/shylu/shylu_node/basker/src/shylubasker_nfactor_blk.hpp

```diff
@@ -1487,6 +1520,9 @@ namespace BaskerNS
     
     //L.col_ptr[k-bcol+1] = lnnz;
     L.col_ptr(k+1) = lnnz;
+    #ifdef BASKER_DEBUG_NFACTOR_BLK
+    if (blkcol == 2 && blkrow == 1) printf( " L.colptr(%d) = %d\n",k+1,lnnz );
```

Comment:
Note for later cleanup (future PR if preferred), assuming that the specific blkcol and blkrow numbers were tied to a matrix being tested